### DISCUSSION TITLE
model context protocol (mcp)

### DISCUSSION THREAD

**Discussion by nikhilmaddirala** [logancyang/obsidian-copilot#965] [Ideas]
I'd love to see support for MCP in Obsidian Copilot. That's the main thing preventing me from switching away from the Claude app!

**Comment by logancyang** [comment]
@nikhilmaddirala Thanks for the suggestion! Could you share some use cases, how you would use MCP in Copilot?

**Reply by nikhilmaddirala** [reply]
For my personal use cases, here are some examples of mcp servers I use for research and personal knowledge management:

- [mzxrai/mcp-webresearch](https://github.com/mzxrai/mcp-webresearch) 🔍📚 - Search Google and do deep web research on any topic
- [@modelcontextprotocol/server-google-drive](https://github.com/modelcontextprotocol/servers/tree/main/src/gdrive) 📇 ☁️ - Google Drive integration for listing, reading, and searching files
- [@modelcontextprotocol/server-filesystem](https://github.com/modelcontextprotocol/servers/tree/main/src/filesystem) 📇 🏠 - Direct local file system access.
- [@modelcontextprotocol/server-brave-search](https://github.com/modelcontextprotocol/servers/tree/main/src/brave-search) 📇 ☁️ - Web search capabilities using Brave's Search API
- [MarkusPfundstein/mcp-gsuite](https://github.com/MarkusPfundstein/mcp-gsuite) - 🐍 ☁️ - Integration with gmail and Google Calendar.

For broader use cases, check out this list of awesome mcp servers: https://github.com/punkpeye/awesome-mcp-servers
